cmd/main_test.go

```diff
@@ -85,8 +85,8 @@ func TestOptions_Validate_FromFlags(t *testing.T) {
 		},
 		{
 			name:     "rerun-fails with failfast",
-			args:     []string{"--rerun-fails", "--packages=./...", "--", "-failfast"},
-			expected: "-failfast can not be used with --rerun-fails",
+			args:     []string{"--rerun-fails", "--packages=./...", "--", "-test.failfast"},
+			expected: "-test.failfast can not be used with --rerun-fails",
```

Comment:
```suggestion
			args:     []string{"--rerun-fails", "--packages=./...", "--", "-failfast"},
			expected: "-failfast can not be used with --rerun-fails",
		},
		{
			name: "rerun-fails with test.failfast",
			args:     []string{"--rerun-fails", "--packages=./...", "--", "-test.failfast"},
			expected: "-test.failfast can not be used with --rerun-fails",
```

Reply:
ahh good call, tested locally as well I took the return from go test for what flags to use literally. Hope the change works